compiler/rustc_middle/src/ty/closure.rs

```diff
@@ -56,12 +56,12 @@ pub enum UpvarCapture {
     ByRef(BorrowKind),
 }
 
-pub type UpvarListMap = FxHashMap<DefId, FxIndexMap<hir::HirId, UpvarId>>;
-pub type UpvarCaptureMap = FxHashMap<UpvarId, UpvarCapture>;
+pub type UpvarListMap = FxIndexMap<DefId, FxIndexMap<hir::HirId, UpvarId>>;
+pub type UpvarCaptureMap = FxIndexMap<UpvarId, UpvarCapture>;
```

Comment:
Those type aliases are not used anywhere, are they?

Reply:
No, they aren't. I can remove them if you think we should.